### PR TITLE
Add Connect4 Game

### LINKED ISSUE #480: Connect-Four game command

### PR DESCRIPTION
**Pull request by fernando457829**
closes #480
depends #1246 

# Prints

Screenshot_20201025_211431
Screenshot_20201025_211541

Yeah, I'm really bad on this game :cry:

### CODE REVIEW

**Review by Doges — APPROVED** [review]
LGTM

**Comment by almeidx** [comment]
conflicts

File: src/loaders/LocaleLoader.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ module.exports = class LocaleLoader extends Loader {
 
       try {
         i18next.use(translationBackend).init({
-          ns: [ 'categories', 'commands', 'commons', 'errors', 'music', 'permissions', 'regions', 'moderation', 'lolservers', 'languages', 'countries' ],
+          ns: [ 'categories', 'commands', 'commons', 'errors', 'music', 'permissions', 'regions', 'moderation', 'lolservers', 'languages', 'countries', 'game' ],
```
Comment (metehus) [inline src/loaders/LocaleLoader.js:46]: wouldn't it be better "games"?

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -1913,6 +1913,11 @@
       }
     }
   },
+  "connect4": {
+    "commandDescription": "Challenge someone to a game of Connect4.",
+    "commandUsage": "<user to challenge>",
+    "missingUser": "You have to give me someone to challenge!"
```
Comment (metehus) [inline src/locales/en-US/commands.json:1919]:
```suggestion
    "missingUser": "You must give me someone to challenge!"
```

Comment (Doges) [reply src/locales/en-US/commands.json:1919]: https://github.com/SwitchbladeBot/switchblade/pull/1246

File: src/loaders/LocaleLoader.js (hunk 1 above)
Comment (Doges) [reply src/loaders/LocaleLoader.js:46]: https://github.com/SwitchbladeBot/switchblade/pull/1246

**Review by almeidx — DISMISSED** [review]

**Comment by almeidx** [comment]
conflicts, once again

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**